docs/linux/external_fuzzing_usb.md

```diff
@@ -216,9 +201,11 @@ cp bin/linux_arm/syz-executor ~/syz-bin/
     ``` bash
     $ cat usb.log
     r0 = syz_usb_connect(0x0, 0x24, &(0x7f00000001c0)={{0x12, 0x1, 0x0, 0x8e, 0x32, 0xf7, 0x20, 0xaf0, 0xd257, 0x4e87, 0x0, 0x0, 0x0, 0x1, [{{0x9, 0x2, 0x12, 0x1, 0x0, 0x0, 0x0, 0x0, [{{0x9, 0x4, 0xf, 0x0, 0x0, 0xff, 0xa5, 0x2c}}]}}]}}, 0x0)
-    $ sudo ./syz-bin/syz-execprog -executor ./syz-bin/syz-executor -threaded=0 -collide=0 -procs=1 -enable='' -debug usb.log
+    $ sudo ./syz-bin/syz-execprog -slowdown 1 -executor ./syz-bin/syz-executor -threaded=0 -collide=0 -procs=1 -enable='' -debug usb.log
```

Comment:
Isn't `-slowdown 1` the default?

Reply:
Yes, it is, but I am not sure what kind value to recommend for fuzzing external targets. That's why I just hinted for this parameter.

Reply:
Well, since the old fix-up patch changed timeouts from 2000 to 5000, I think it makes sense to recommend slowdown 3.

Does it work for you with slowdown 1?

Reply:
Yes, it worked with single tests on USB Armory but I haven't done extensive fuzzing yet.

I will change it to 3.

Reply:
Sounds good. I'm planning to retest everything soon, so I'll fix it in case we need a different slowdown.